tensorflow_recommenders_addons/dynamic_embedding/core/kernels/cuckoo_hashtable_op.cc

```diff
@@ -115,6 +155,57 @@ struct LaunchTensorsInsert<CPUDevice, K, V> {
   const int64 value_dim_;
 };
 
+template <typename Device, class K, class V>
+struct LaunchTensorsAccum;
+
+template <class K, class V>
+struct LaunchTensorsAccum<CPUDevice, K, V> {
+  explicit LaunchTensorsAccum(int64 value_dim) : value_dim_(value_dim) {}
+
+  void launch(OpKernelContext* context, cpu::TableWrapperBase<K, V>* table,
+              const Tensor& keys, const Tensor& values_or_deltas,
+              const Tensor& exists) {
+    const auto key_flat = keys.flat<K>();
+    int64 total = key_flat.size();
+    const auto values_or_deltas_flat = values_or_deltas.flat_inner_dims<V, 2>();
+    const auto exist_flat = exists.flat<bool>();
+
+    auto shard = [this, &table, key_flat, &values_or_deltas_flat, &exist_flat](
+                     int64 begin, int64 end) {
+      for (int64 i = begin; i < end; ++i) {
+        if (i >= key_flat.size()) {
+          break;
+        }
+        table->insert_or_accum(key_flat(i), values_or_deltas_flat,
+                               exist_flat(i), value_dim_, i);
+      }
+    };
+    auto& worker_threads = *context->device()->tensorflow_cpu_worker_threads();
+    // Only use num_worker_threads when
+    // TFRA_NUM_WORKER_THREADS_FOR_LOOKUP_TABLE_INSERT env var is set to k where
+    // k > 0 and k <current number of tf cpu worker threads. Otherwise nothing
+    // changes.
+    int64 num_worker_threads = -1;
+    Status status =
+        ReadInt64FromEnvVar("TFRA_NUM_WORKER_THREADS_FOR_LOOKUP_TABLE_INSERT",
```

Comment:
Is there no need to obtain the environment variable TFRA_NUM_WORKER_THREADS_FOR_LOOKUP_TABLE_INSERT?

TFRA_NUM_WORKER_THREADS_FOR_LOOKUP_TABLE_INSERT is to start serving to adjust the number of threads to control the insert speed.

Reply:
Ok, I removed it